docs/roadmap.rst

```diff
@@ -35,14 +35,14 @@ This section will specify the environment for the development of libCellML.
 * Test: unit testing to use `gtest <https://github.com/google/googletest/>`_.
 * Documentation: written in `reStructuredText <http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html>`_.
 * Documentation: API and source code examples will be documented using C++-style `Doxygen <http://www.doxygen.org/>`_ comments.
-* Website: `libcellml.org <https://libcellml.org>`_.
+* Website: `libcellml.org <https://libcellml.org>`__.
```

Comment:
seems like this change should be all over the place :) https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use (but I assume this one change was made for a specific reason and we can leave it there for now)

Reply:
We do receive warnings about duplicated targets so as long as we keep our documentation warnings to zero we should be fine.  If someone embeds our documentation into theirs?? then an issue would have to be created to get us to fix the target name.